docs/products/databases/managed-databases/guides/create-database/index.md

```diff
@@ -48,6 +49,10 @@ Every node of a database cluster is built on its own Linode Compute Instance. In
 
 - [Choosing a Compute Instance Type and Plan](/docs/guides/choosing-a-compute-instance-plan/)
 
+{{< note >}}
+Once a Managed Database cluster is created, it cannot be resized to a different plan. To modify the resources allocated to your database, you will need to create a new Managed Database with the desired plan, migrate your data, and delete the original Managed Database.
```

Comment:
am I overthinking this, but maybe it would be beneficial to link to the migrate database guide here
https://www.linode.com/docs/products/databases/managed-databases/guides/migrate-mysql/

And a link to a deleting a database guide if we have that (not finding one right now)

Reply:
Thinking about this now, we don't have a single migration guide. We have one for each database engine, so might be too verbose to link to each one. Since this lives directly alongside those guides (so they should already be visible in the explore docs menu), going to leave out the links for now.